LOADING_SUPPORT.md

```diff
@@ -0,0 +1,186 @@
+   `known_supported_records_formats_for_load()` method.  For now we're
+   going to assume the very best case - that your database will
+   support every single records format we support--or at least the
+   particularly useful ones.  This will no doubt not be true by the time
+   we get done implementing - but the tests will catch that and we'll
+   tweak this down to what's actually supported by the time we're
+   done.  No need to implement anything else yet.
+
+   ```python
+   return [
+       ParquetRecordsFormat(),
+       DelimitedRecordsFormat(variant='bluelabs'),
+       DelimitedRecordsFormat(variant='csv'),
+       DelimitedRecordsFormat(variant='bigquery'),
+       DelimitedRecordsFormat(variant='vertica'),
+   ]
+   ```
+
+5. Run the test and start addressing issues one by one by implementing
+   the function in question.  This will be slow, finicky work - do
+   yourself a favor and leave yourself copious notes as comments
+   pointing back to the original online documentation for the database
+   in question as you go.  Follow the other drivers' use of
+   `cant_handle_hint()` and `quiet_remove()`.  The MySQL loader in
+   particular uses hint typing to ensure we are covering cases
+   correctly, and has written types for the valid input into MySQL's
+   LOAD statement as well.
+
+   To get the test to pass, you may need to tweak the formats slightly
+   (e.g., changing date formats to ones supported or specifying no
+   compression).  You may not be able to support certain variants or
+   formats at all - leave a comment in the method documenting all of
+   these situations and describing why.
+
+6. Write a unit test for `can_load_this_format()` - see
+   `tests/unit/db/mysql/test_mysql_db_driver.py` for an example.  Now
+   implement `can_load_this_format()` in a similar way to other
+   drivers and verify the tests pass.
+
+7. Figure out which functions are left raising NotImplementedError, if
+   any.  Write a bunch of T ODO comments documenting the different
+   cases that need to be handled.  Include cases for invalid
+   combinations (expect them to raise exceptions).  Write a test for
+   the first one.  Implement enough to get that test to pass, and then
+   lather, rinse and repeat.
+
+8. Get a sample load working with `mvrec`.  Having your
+   already-working proof of concept to cross debug with will be vital
+   at this point, unless you're one of those lucky engineers for whom
+   everything works the first time...
+
+9. Run `./itest` with your database target and work through any issues
+   found.
+
+10. Get unit tests passing by updating to match whatever changes you
+    needed to make for the integration tests.
+
+11. Increase unit test coverage to match or exceed the current high
+    water mark.  You can look at `cover/index.html` and sort by
+    'missing' to see likely reasons why coverage has slipped.
+
+[done to here]
+
+TODO: Update mover_test_case.py
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker